### ISSUE TITLE
[Question] Is Dropout enabled in offline experiments?

### ISSUE THREAD

**Issue by typoverflow** [Div99/XQL#4]
Hi, 
Thanks for open-sourcing the code! I'm quite new to jax and I have one question about the dropout on value networks in offline experiments. It seems that whenever you call the value network (https://github.com/Div99/XQL/blob/main/offline/critic.py), you did not pass in `training` and thus the `training` is set to `False` by default, which disables the dropout on value networks. Is there anything I missed?

**Comment by jhejna** [comment]
I believe dropout is only enabled for Franka Kitchen. We directly borrow the configs and hyper-parameters from the IQL repository (https://github.com/ikostrikov/implicit_q_learning)

**Comment by typoverflow** [comment]
I see, thanks :)
